public/app.js

```diff
@@ -0,0 +1,233 @@
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
You should add this to your .gitignore I think, since it's just a transpilation of everything else.